sys/net/crosslayer/ng_netreg/ng_netreg.c

```diff
@@ -101,4 +102,30 @@ ng_netreg_entry_t *ng_netreg_getnext(ng_netreg_entry_t *entry)
     return entry;
 }
 
+ng_pktsnip_t *ng_netreg_hdr_build(ng_nettype_t type, ng_pktsnip_t *payload,
+                                  uint8_t *src, uint8_t src_len,
+                                  uint8_t *dest, uint8_t dest_len)
+{
+    switch (type) {
+#ifdef MODULE_NG_IPV6
+
+        case NG_NETTYPE_IPV6:
+            return ng_ipv6_hdr_build(payload, src, src_len, dest, dest_len);
```

Comment:
This seems to rely on ``sys/include/net/ng_ipv6/hdr.h``, which isn't in the master yet, but https://github.com/RIOT-OS/RIOT/pull/2454, which includes this file, relies on this PR... I'm sure that's according to your plans, but I'm a bit confused.